packages/base/src/widget.ts

```diff
@@ -82,6 +82,17 @@ export class WidgetModel extends Backbone.Model {
     };
   }
 
+  generateMimeBundle() {
+    return Promise.resolve({
+      'application/vnd.jupyter.widget-view+json': {
+        model_id: this.model_id,
+        version_major: 2,
```

Comment:
Should we take the version from what the model defines instead?

Reply:
I forgot to mention that this should be removed, actually

Reply:
Oh ok :) 👍🏼